src/clojurebot/core.clj

```diff
@@ -85,12 +90,16 @@
 (reg-command
   "doc"
   (fn [sym-name]
-    (if sym-name
+    (if (not (empty? sym-name))
```

Comment:
Why are you checking for empty? Its not supposed to be a collection, its a symbol. Right?